package.json

```diff
@@ -17,10 +17,12 @@
     "test:watch": "karma start",
     "test:watch:chrome": "karma start --browsers Chrome",
     "test:server": "http-server ./ -a localhost -p 8080 -c-1",
-    "testcafe": "testcafe",
+    "testcafe": "EDGE_ENV=\"'ee'\" testcafe",
```

Comment:
This is the environment but should we call this EDGE_BASE_PATH instead to be more consistent?

Reply:
@shammowla Let's merge, and add a TODO to maybe rename this VAR. Thanks @liljenback